resources/assets/js/tippy.js

```diff
@@ -29,6 +29,7 @@ const initTippy = (parentEl = document.body) => {
         if (el.getAttribute("data-tippy-hover")) {
             instanceSettings.touch = "hold";
             instanceSettings.trigger = "mouseenter";
+            instanceSettings.allowHTML = true;
```

Comment:
not sure I like this as a general setting, as this may introduce XSS issues when it is used as a tooltip to show user content somewhere and generally we don't need HTML support